src/home/ActionsCarousel.tsx

```diff
@@ -120,6 +121,9 @@ function ActionsCarousel() {
 }
 
 const styles = StyleSheet.create({
+  container: {
+    zIndex: -1,
```

Comment:
This was for the info bubble next to the wallet balance for dapp positions; it would appear underneath the quick actions, and only changing the zIndex of the info bubble (but not the quick actions) appeared to have no effect. fortunately since we're getting rid of the info bubble, we can remove this hack!